imod/tests/test_mf6/test_mf6_simulation.py

```diff
@@ -479,9 +479,12 @@ def test_import_from_imod5(imod5_dataset, tmp_path):
         default_simulation_distributing_options,
     )
 
-    simulation["imported_model"] = OutputControl(save_head="last", save_budget="last")
+    simulation["imported_model"]["output"] = OutputControl(
+        save_head="last", save_budget="last"
+    )
 
     simulation.create_time_discretization(["01-01-2003", "02-01-2003"])
 
-    # write and validate the simulation
-    simulation.write(tmp_path, False, True, False)
+    # write and validate the simulation.
+    simulation.write(tmp_path, binary=False, validate=True)
+    pass
```

Comment:
Please remove the ``pass`` statement. (I guess a relict of the debugging session?)